examples/sorted-set-example.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+require "vendor/autoload.php";
+
+use Momento\Auth\CredentialProvider;
+use Momento\Cache\CacheClient;
+use Momento\Config\Configurations\Laptop;
+use Momento\Logging\StderrLoggerFactory;
+use Momento\Requests\CollectionTtl;
+use Psr\Log\LoggerInterface;
+
+$CACHE_NAME = uniqid("php-sorted-set-example-");
+$SET_NAME = "example-sorted-set";
+$ITEM_DEFAULT_TTL_SECONDS = 60;
+
+// Setup
+$authProvider = CredentialProvider::fromEnvironmentVariable("MOMENTO_API_KEY");
+$configuration = Laptop::latest(new StderrLoggerFactory());
+$client = new CacheClient($configuration, $authProvider, $ITEM_DEFAULT_TTL_SECONDS);
+$logger = $configuration->getLoggerFactory()->getLogger("ex:");
+
+function printBanner(string $message, LoggerInterface $logger): void
+{
+    $line = "******************************************************************";
+    $logger->info($line);
+    $logger->info($message);
+    $logger->info($line);
+}
+
+printBanner("*                      Momento Example Start                     *", $logger);
+
+// Ensure test cache exists
+$response = $client->createCache($CACHE_NAME);
+if ($response->asSuccess()) {
+    $logger->info("Created cache " . $CACHE_NAME . "\n");
+} elseif ($response->asError()) {
+    $logger->info("Error creating cache: " . $response->asError()->message() . "\n");
+    exit(1);
+} elseif ($response->asAlreadyExists()) {
+    $logger->info("Cache " . $CACHE_NAME . " already exists.\n");
+}
+
+// The CollectionTtl object is used to control TTLs for collections as they
+// are updated. By default, it is configured to reset the TTL for the collection
+// to the client's default TTL each time the collection is updated.
+$collectionTtl = new CollectionTtl();
+
+// add element to sorted set
+$logger->info("Adding element 'one' to sorted set $SET_NAME\n");
+$response = $client->sortedSetPutElement($CACHE_NAME, $SET_NAME, "one", 1.0, $collectionTtl);
+if ($response->asSuccess()) {
+    $logger->info("SUCCESS: Added element to set\n");
+} elseif ($err = $response->asError()) {
+    $logger->info("Error adding element to set: {$err->message()}\n");
+    exit(1);
+}
+
+// add multiple elements to sorted set
+$logger->info("Adding elements 'two', 'three', 'four', 'five' to sorted set $SET_NAME\n");
+$response = $client->sortedSetPutElements($CACHE_NAME, $SET_NAME, [
+    "two" => 2.0,
+    "three" => 3.0,
+    "four" => 4.0,
+    "five" => 5.0
+], $collectionTtl);
+if ($response->asSuccess()) {
+    $logger->info("SUCCESS: Added elements to set\n");
+} elseif ($err = $response->asError()) {
+    $logger->info("Error adding elements to set: {$err->message()}\n");
+    exit(1);
+}
+
+// fetch sorted set by rank (0 to -1 for all elements) in ascending order
+$logger->info("Fetching sorted set $SET_NAME\n");
+$response = $client->sortedSetFetchByRank($CACHE_NAME, $SET_NAME, 0, -1);
```

Comment:
Why not leave off the ranks for fetching all elements?

Reply:
I am looking into that. It was giving me error saying:
```
[ex:]: Error fetching sorted set: Invalid argument passed to Momento client: minScore must be less than or equal to maxScore
```